doc/sphinx/proof-engine/ltac.rst

```diff
@@ -2340,7 +2339,7 @@ performance issue.
    Set Ltac Profiling.
    tac.
    Show Ltac Profile.
-   Show Ltac Profile "omega".
+   Show Ltac Profile "lia".
```

Comment:
@JasonGross No objection to this example using `lia` now?

Reply:
No objection